packages/components/src/TestComponent/TestComponent.spec.tsx

```diff
@@ -0,0 +1,6 @@
+import * as TestComponentStoryFile from "./TestComponent.stories";
+import { generateSnapshots } from "@chanzuckerberg/story-utils";
+
+describe("<TestComponent />", () => {
+  generateSnapshots(TestComponentStoryFile);
+});
```

Comment:
This is a file generated from the script for the purpose of reviewing. TODO: remove this file before merging this PR.